### DISCUSSION TITLE
Images from the clipboard or url

### DISCUSSION THREAD

**Discussion by Lorg0n** [Comfy-Org/ComfyUI#1218] [Ideas]
Why not make it so that if there is a picture in the buffer exchange now, then by pressing CTRL + V keys a node is created and the picture is automatically loaded into temporary files.

Or so that in the "Load Image" node there would be a field where you could insert a link to an image on the Internet.

**Comment by toyxyz** [comment]
It would be very convenient if the user could copy the image to a clipboard -Ctrl+C in the preview image node and paste the clipboard image with Ctrl+V in the Load image node.

**Reply by Lorg0n** [reply]
It would have saved me a lot of time

**Comment by basvdelsen** [comment]
Being able to copy paste images from the internet into comfyui without having to save them, and copying from comfyui into photoshop and vice versa without having to save the pictures, these would be really nice.

**Comment by Subby** [comment]
One of the features I miss from A1111 for sure! It's just so convenient

**Reply by karl-von-knecht** [reply]
yes, please! Maybe it does not have to go into the comfyui core, maybe someone can make a node with the single purpose of copying an image to the clipboard.

**Comment by comfyanonymous** [comment]
Copy pasting images to the UI should work on the latest version.

**Reply by Subby** [reply]
Thanks, had no idea! Will give it a shot

**Reply by Subby** [reply]
Can confirm it's working in the "Load Image" node, thank you very much.

**Comment by coreyryanhanson** [comment]
For the URL loading, there are a few custom nodes that already do that.
One here https://github.com/WASasquatch/was-node-suite-comfyui/tree/main
Another here https://github.com/sipherxyz/comfyui-art-venture/tree/main

**Comment by Ulf3000** [comment]
copy and paste is my favourite workflow from clipstudio to stable diffusion and back
